.storybook/preview.js

```diff
@@ -129,6 +130,8 @@ const options = {
   ],
 };
 
+window.FormUtils = FormUtils;
```

Comment:
Not sure that we need it?  I need your opinion.

It is really rather functionality targeted to developers so I do not want to expose it to other projects like record-manager-ui. Is this a standard (best practice) way how to solve such an issue? 

I can imagine 2 situations in how to use it:
- for developer --  debugging the form 
- for application that uses the library -- it can show specification the form to the user if form rendering fails. This would have to be think through -- so let's not discuss it here.

So my question is, is it impossible to use in the development console without exposing it to the `window`? I would rather extend development notes within the SForms than pollute the `window`.
Moreover, polluting it with FormUtils is strange in sense that most of the methods in FormUtils are not useful for the developer, or at least as useful as e.g. JsonLdUtils, which we do not expose ...

Reply:
Without exposing it to window, we cannot use it "easily" as a script in the console. The alternatives I see are:
- Putting it in a different javascript and run it with something like `node PrintFormSpecification.js`. However, it is hard in javascript to run a specific function from CLI (we would have to add some tooling, according to [this post](https://stackoverflow.com/questions/30782693/run-function-in-script-from-command-line-node-js), the main issue is that we are using a `module`). So we would have to run it as it is as change the questions directly inside the JS file
- Write a bash script

However, to run from the console I do not see any other alternative but to expose it to the window. We could create another JS file and expose only this specific file and function

Reply:
It is now working in the toolbar of storybook: 
![image](https://github.com/kbss-cvut/s-forms/assets/57750426/00c40fa2-e654-4181-9c16-3926cd0db95f)

Reply:
Looks great !!! It can also be copied to clipboard (although i have to select it all) so it is very useful.

However, i see weird output there, which i suggest to formulate as new issue while closing this one:
![image](https://github.com/kbss-cvut/s-forms/assets/3962237/ebc116e1-95b7-43ed-a00c-1dd3259605d8)

Reply:
Ticket was created here: https://github.com/kbss-cvut/s-forms/issues/310